.github/workflows/approve-snapshots.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         run: pnpm build
 
       - name: Serve ⚡️
-        run: pnpm serve & pnpx wait-on http://localhost:5173
+        run: pnpm serve & pnpm dlx wait-on http://localhost:5173
```

Comment:
`pnpx` is not actually documented, so I wonder if they're planning to drop it. To be safe, I'm switching to the more verbose command.